packages/@divvi/mobile/src/send/SendConfirmation.tsx

```diff
@@ -88,9 +88,15 @@ export default function SendConfirmation(props: Props) {
   )
 
   useEffect(() => {
+    // do not refresh if prepared transactions is available in the route params
+    if (!fromExternal) {
```

Comment:
```suggestion
    if (props.route.params.prepareTransactionsResult) {
```
maybe allows you to remove the comment